src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -243,6 +247,14 @@ export default {
 		margin-left: calc(-1 * min($navigation-width, var(--app-navigation-max-width)));
 	}
 
+	&__search {
+		width: 100%;
+	}
+
+	&__body {
+		overflow-y: scroll;
```

Comment:
What's the idea behind this? It causes scrolling in Mail: 
![image](https://github.com/user-attachments/assets/fab1c507-8139-46a0-8311-bde4e4ab3e1e)

Reply:
replace `<template #default>` with `<template #search>`

Reply:
> replace `<template #default>` with `<template #search>`

Only if for the search component